web/src/app/components/Game.tsx

```diff
@@ -32,33 +37,38 @@ export default function Game({roomID}: GameProps) {
     useEffect(() => {
 
         socket.on('connect', () => {
-            setConnected(true);
             currPlayerID.current = socket.id as string;
             const username = sessionStorage.getItem('username') ?? createGuestName()
             socket.emit('joinRoom', roomID, username)
         });
 
         socket.on('disconnect', () => {
-            setConnected(false);
-            setStarted(false);
+            disconnect();
```

Comment:
Where is this function?

Reply:
line 120. I'll rename this since we already have the disconnect event as well as socket.disconnect. It's a bit confusing